pylint/checkers/refactoring/recommendation_checker.py

```diff
@@ -345,6 +345,10 @@ def _detect_replacable_format_call(self, node: nodes.Const) -> None:
             if isinstance(node.parent.parent, nodes.Assign):
                 return
 
+            # If the format function is not called, ignore this
+            if not isinstance(node.parent.parent, nodes.Call):
```

Comment:
I'm wondering if we can't merge this with the check on L345. If the check on L349 passes, L345 should pass as well. We would only need to add both use cases to the comment on L344.